docs/topics/wasm/wasm-overview.md

```diff
@@ -1,61 +1,75 @@
 [//]: # (title: Kotlin Wasm)
 
 > Kotlin Wasm is [Alpha](components-stability.md).
-> It may be changed at any time. Use it only for evaluation purposes.
+> It may be changed at any time. You can use it in scenarios before production. We would appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
 >
-> We would appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
+> [Join the Kotlin/Wasm community](https://slack-chats.kotlinlang.org/c/webassembly).
 >
 {type="note"}
 
-[WebAssembly (Wasm)](https://webassembly.org) is a binary instruction format for a stack-based virtual machine.
-This format is platform-independent because it runs on its own virtual machine.
-Wasm is designed to be fast and secure, and it can compile code from various programming languages, including Kotlin.
+Kotlin has the power to build applications, and reuse mobile and desktop user interfaces (UIs) in your web projects with
+Compose Multiplatform for Web and Kotlin/Wasm.
 
-Kotlin/Wasm is a new compilation target for Kotlin. You can use it in your Kotlin Multiplatform projects.
-With Kotlin/Wasm, you can create applications that run on different environments and devices supporting WebAssembly and meeting Kotlin's requirements.
+[Compose Multiplatform]((https://www.jetbrains.com/lp/compose-multiplatform/)) is a declarative framework based on Kotlin
+and [Jetpack Compose](https://developer.android.com/jetpack/compose), which allows you to implement the UI
+once and share it across all the platforms you target. Specifically for web platforms, Compose Multiplatform for Web uses 
+Kotlin/Wasm as its compilation target.
 
-> Learn more about Kotlin/Wasm with this [YouTube playlist](https://kotl.in/wasm-pl).
+Explore our online demo of an application built with Compose Multiplatform for Web and Kotlin/Wasm:
+
+<a href="https://zal.im/wasm/jetsnack/"><img src="wasm-demo.png" width="700" alt="Kotlin/Wasm demo"/></a>
+
+>To run applications built with Kotlin/Wasm in a browser, you need a browser version that supports the new garbage collection
+> and exception handling proposals. To check the support status, see the [WebAssembly
+> roadmap](https://webassembly.org/roadmap/).
 >
-{type="note"}
+{type="tip"}
 
-## Browser support
+[WebAssembly (Wasm)](https://webassembly.org/) is a binary instruction format for a stack-based virtual machine. This
+format is platform-independent because it runs on its own virtual machine. Wasm provides languages like Kotlin, and many 
+more, with a compilation target so that they can run on the web.
```

Comment:
I found this one (the last sentence) a bit hard to read. 
But I'm not sure, maybe it is only me. WDYT about it?

Reply:
Changed it to "Wasm provides Kotlin and other languages with a compilation target to run on the web". Clearer?